src/components/ec-container/ec-container.vue

```diff
@@ -1,50 +1,46 @@
 <template>
   <div class="ec-container">
-    <template v-if="title">
-      <h1 class="ec-container__title">{{ title }}</h1>
-      <p class="ec-container__title-intro">{{ titleIntro }}</p>
-    </template>
-    <slot />
+    <div
+      class="ec-container__navigation"
+      :class="{ 'ec-container__navigation--can-be-collapsed': canBeCollapsed }"
+    >
+      <slot name="navigation" />
+    </div>
+    <div class="ec-container__content">
+      <slot name="content" />
+    </div>
   </div>
 </template>
 
 <script>
 export default {
   name: 'EcContainer',
   props: {
-    title: {
-      type: String,
-      default: null,
-    },
-    titleIntro: {
-      type: String,
-      default: null,
+    canBeCollapsed: {
+      type: Boolean,
+      default: false,
     },
   },
 };
 </script>
 
 <style lang="scss">
-@import '../../scss/tools/typography';
-@import '../../scss/settings/colors/gray-color-scale';
-
 .ec-container {
-  padding: 1.5rem;
+  display: flex;
+  flex-direction: row;
+  min-height: 100vh;
 
-  &__title {
-    @include h1;
+  &__navigation {
+    position: relative;
+    flex-basis: 280px;
 
-    margin-top: 0;
-    margin-bottom: 0.5rem;
+    &--can-be-collapsed {
```

Comment:
`isCollapsable`? Also I never remember if in CSS we'll go with `isCollapsable` or `is--collapsable`